src/Applications/RTFRevit/ConsoleOutInterceptor.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RTF.Applications
+{
+    public interface IConsoleTraceListener
+    {
+        void OnConsoleOutLine(string text);
+
+        void OnErrorOutLine(string text);
+    }
+
+    public class ConsoleOutInterceptor : IDisposable
```

Comment:
can you add summary tags for this public class and public methods?

Reply:
fixed